src/main/java/com/github/hey_world_team/currency_converter/service/ConversionService.java

```diff
@@ -0,0 +1,97 @@
+package com.github.hey_world_team.currency_converter.service;
+
+import com.github.hey_world_team.currency_converter.model.BestConversion;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.*;
+
+@Service
+public class ConversionService {
+
+    //TODO replace these maps with real repository
+    //temporary solution for storing data about cryptocurrencies
+    private final Map<String, BigDecimal> cryptoRatesToUsd;
+    private final Map<String, BigDecimal> cryptoRatesToRub;
+    private final Map<String, BigDecimal> cryptoRatesToEur;
+
+    @Autowired
+    public ConversionService(CurrencyService service) {
+        // Rub 100000
+        cryptoRatesToUsd = new LinkedHashMap<>();
+        cryptoRatesToUsd.put("USDT", new BigDecimal("1.01")); // Курс USDT к USD
```

Comment:
need to translate russian comments

Reply:
also, i think  we don't need this part

if the cost of cryptocurrencies in source currency

our path next:

we have source currency -> crypto cost of source currency -> target currency in the request

so, we need to take from the repository cost of 1 target currency by  source currency

also, we need to prepare the correct cost for every crypto by the amount of source currency

then we need to compare their